<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QDialog" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="RoomBG_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
    background-color: skyblue;
}
</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="LeftSide_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
    background-color: #2c3e50;  /* Dark blue-gray */
    color: white;               /* Text color */
    font-weight: bold;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="DashName_label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DASHBOARD</string>
   </property>
  </widget>
  <widget class="QToolButton" name="overviewButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roomButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rooms</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reserveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve Room</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maintenanceButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QToolButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>971</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="OccupiedRooms">
    <attribute name="title">
     <string>Occupied Rooms</string>
    </attribute>
    <widget class="QTableWidget" name="occupiedRoomsTable">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>861</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
 color: red;
gridline-color: blue;
selection-color: green;
selection-background-color: blue;
font: italic 14pt &quot;Times New Roman&quot;;
}
QTableWidget&quot;&quot;item:selected
{
color: white;
background-color: red;
}
QheaderView {
background-color:black;
color: white;
font: italic 14pt &quot;Times New Roman&quot;;
}</string>
     </property>
     <column>
      <property name="text">
       <string>Room No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Student ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Capacity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration of stay</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="AvailabeRooms">
    <attribute name="title">
     <string>Available Rooms</string>
    </attribute>
    <widget class="QTableWidget" name="availableRoomsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>861</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
 color: red;
gridline-color: blue;
selection-color: green;
selection-background-color: blue;
font: italic 14pt &quot;Times New Roman&quot;;
}
QTableWidget&quot;&quot;item:selected
{
color: white;
background-color: red;
}
QheaderView {
background-color:black;
color: white;
font: italic 14pt &quot;Times New Roman&quot;;
}</string>
     </property>
     <column>
      <property name="text">
       <string>Room No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Capacity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="DamagedRooms">
    <attribute name="title">
     <string>Damaged Rooms</string>
    </attribute>
    <widget class="QTableWidget" name="damagedRoomsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>861</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
 color: red;
gridline-color: blue;
selection-color: green;
selection-background-color: blue;
font: italic 14pt &quot;Times New Roman&quot;;
}
QTableWidget&quot;&quot;item:selected
{
color: white;
background-color: red;
}
QheaderView {
background-color:black;
color: white;
font: italic 14pt &quot;Times New Roman&quot;;
}</string>
     </property>
     <column>
      <property name="text">
       <string>Room No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Capacity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Repair Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>Search Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="search_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Student ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
